src/app/beer_garden/config.py

```diff
@@ -733,7 +733,7 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
                     "type": "bool",
                     "default": False,
                     "description": "Publish events to parent garden over HTTP",
-                    "alt_env_names": ["PARENT_HTTP"],
+                    "alt_env_names": ["PARENT_HTTP_ENABLE"],
```

Comment:
We don't need to specify this here - "PARENT_HTTP_ENABLE" *is* the environment name.